src/Base/Geometry.cpp

```diff
@@ -14,11 +18,24 @@ void init_Geometry(py::module& m)
 {
     py::class_<Geometry, CoordSys>(m, "Geometry")
         .def("__repr__",
-             [](const Geometry&) {
-                 return "<amrex.Geometry>";
-             }
+             [](py::object& obj) {
+                 py::str py_name = obj.attr("__class__").attr("__name__");
+                 const std::string name = py_name;
+                 const auto gm = obj.cast<Geometry>();
+                 std::stringstream s;
+                 s << gm;
+                 return "<amrex." + name + " " + s.str() + ">";
+            }
         )
+        .def("__str",
```

Comment:
I think this is called `__str__`:
```suggestion
        .def("__str__",
```

Would it be sufficient if we implement `__repr__` for now or did you have a specific use case in mind when you added this?
https://stackoverflow.com/questions/1436703/what-is-the-difference-between-str-and-repr

I mean if `<<` looks reasonable in practice then sure let's go for it. I honestly have not thought about it.

Reply:
I don't have any specific need for `__str__` over `__repr__`, I just followed the `Dim3` and `IntVect` examples without thinking much about it